docs/ssh.md

```diff
@@ -41,6 +41,7 @@ and respects the following options:
 * `Host` (including wildcard matching)
 * `UserKnownHostsFile`
 * `IdentitiesOnly`
+* `BindAddress`
```

Comment:
```suggestion
* `BindAddress` (with the `libssh` backend only)
```

The libssh2 backend doesn't support this, so we should clarify.

If you wanted to add support for that, it's a little involved because
Rust's `TcpStream` doesn't provide a way to do that, so you'd need to use a crate like `socket2` to create the socket, then convert that into `TcpStream`.  https://docs.rs/socket2/latest/socket2/struct.Socket.html#examples 

Those changes would be needed around line 290 of `wezterm-ssh/src/sessioninner.rs` if you did want to take a crack at it!

Reply:
I took a stab at it. I don't write much Rust so I can't promise that it's any good 😄

Reply:
I'm not sure how to test the libssh2 changes though. I've tried running `cargo build --no-default-features --features ssh2`, but when I run it it still says it's using `libssh_rs`.